chapter_15_advanced_forms.asciidoc

```diff
@@ -1117,14 +1121,9 @@ real app.
 We have one unit test file for each of our key source code files.  Here's
 a recap of the biggest (and highest-level) one, 'test_views' (the listing
 shows just the key tests and assertions, and your order may vary):
-// CSANAD: Why are we only showing ListViewTest? HomePageTest, NewListTest
 
-// RITA: Why is this in a box? I don't think it needs to be because it's a code block that is discussed in the preceding body text. Did you do this so it would double as the "wrap-up" box that you have at the end of most chapters? It looks a bit strange this way. 
 
 [[what-to-test-in-views]]
-.What to Test in Views
-******************************************************************************
-
```

Comment:
Let's have an intro sentence like:

_By way of a recap, let's see an outline of all the test methods and main assertions in our test_views.  This isn't to say you should copy-paste these exactly, more like a list of things you should at least consider testing_